draft-packages/menu/KaizenDraft/Menu/Menu.tsx

```diff
@@ -48,6 +52,7 @@ const Menu: Menu = props => {
   const {
     align = "left",
     dropdownWidth = "default",
+    autoHide = true,
```

Comment:
Note that this line doesn't do anything, because the default value is not getting passed down to the render function.